kotlinpoet/src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -17,8 +17,11 @@ package com.squareup.kotlinpoet
 
 import kotlin.reflect.KClass
 
+@OptIn(DelicateKotlinPoetApi::class)
 public class LambdaTypeName private constructor(
   public val receiver: TypeName? = null,
+  @property:DelicateKotlinPoetApi("Context receivers are currently a preview feature")
+  public val contextReceivers: List<TypeName> = emptyList(),
```

Comment:
Is this the right place for this, vs a TypeName property like annotations are?

Reply:
Hmm, I suppose so. I guess we can start with this and can always change it later if need be while it's still experimental